Documentation edits (#17509)
netdata/netdata: docs/monitor/view-active-alerts.md

# Alerts tab

Netdata comes with hundreds of pre-configured health alerts designed to notify you when an anomaly or performance issue affects your node or its applications.

## Active tab

From the Active tab you can see all the active alerts in your War Room. You will be presented with a table having information about each alert that is in warning or critical state.

> You can always sort the table by a certain column by clicking on the name of that column, and use the gear icon on the top right to control which columns are visible at any given time.

Reviewer: ```suggestion
You can always sort the table by a certain column by clicking on the name of that column, and using the gear icon on the top right to control which columns are visible at any given time.
```

- Latest and Triggered time values
- The alert's description
- A link to the Netdata Advisor's page about this alert 
- The chart at the time frame that the alert was triggered
- The alert's information: Node name, chart instance, type, component and class
- Configuration section
- Instance values - Node Instances

At the bottom of the panel you can click the green button "View alert page" to open a [dynamic tab](https://github.com/netdata/netdata/blob/master/docs/quickstart/infrastructure.md#dynamic-tabs) containing all the info for this alert in a tab format, where you can also run correlations and go to the node's chart that raised the particular alert.

### Silence an alert

From this tab, the "Silencing" column shows if there is any rule present for each alert, and from the "Actions" column you can create a new [silencing rule](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/notifications.md#silencing-alert-notifications) for this alert, or get help and information about this alert from the [Netdata Assistant](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-assistant.md).

## Alert Configurations tab

> From this tab you can view all the configurations for all potential alerts in your War Room. Each row concerns one alert, and it provides information about it in the rest of the table columns.

Reviewer 2: ```suggestion
From this tab you can view all the configurations for all running alerts in your War Room. Each row concerns one alert, and it provides information about it in the rest of the table columns.

By running alerts we mean alerts that are related to some metric that is or was collected. Netdata may have more alerts pre-configured that aren't applicable to your monitoring use-cases.

```

this came from a chat I was with Costa and @kapantzak, where he made it clear this isn't all the pre-configured alerts just those related to things Cloud is aware of (that were or are being monitored)

pinging @sashwathn in case he has more to share

Note: we will be changing the tab names as per a suggestion from Costa but not sure when they will be done, so we will need to review this then cc/ @kapantzak

Author: If we change the tab names, let's make sure the documentation gets the proper updates _the same day_

I'm doing this arrangement in the Dashboard sections so it is easier to pinpoint what to update. Ideally the one who merges the PR for the tab changes should in turn go and update related docs around it

> You can control which columns are visible by using the gear icon in the right-hand side.

Reviewer: ```suggestion
You can control which columns are visible by using the gear icon on the right-hand side.
```
